cluster-autoscaler/loop/trigger.go

```diff
@@ -43,19 +43,25 @@ type scalingTimesGetter interface {
 	LastScaleDownDeleteTime() time.Time
 }
 
+type provisioningRequestProcessTimeGetter interface {
```

Comment:
We need a comment somewhere to explain why using the processing times based on the injector is a good idea. As discussed offline, on a face value injector should just append unschedulable pods, but there is some complex rationale for why taking ProvReq processing time from the injector will give a good result.

Reply:
Added brief explanation and rationale as comments in main and trigger